CMakeLists.txt

```diff
@@ -15,6 +13,41 @@ file(GLOB_RECURSE SOURCE_CPP ${CMAKE_SOURCE_DIR}/ *.cpp)
 file(GLOB_RECURSE SOURCE_H ${CMAKE_SOURCE_DIR}/ *.h)
 file(GLOB_RECURSE SOURCE_HPP ${CMAKE_SOURCE_DIR}/ *.hpp)
 
+if(${SIMULATE})
+  include(linux-64.cmake)
+  add_definitions(-DSIM_ON)
```

Comment:
it's literally used in line 51 and the VSCode is consstantly compailing that support for CMake versions < 3.5 is being deprecated, so either we fix the whole file or leave it like this. I think for now it can stay this way